Doc/library/sqlite3.rst

```diff
@@ -222,14 +222,24 @@ Module functions
 
 .. function:: complete_statement(statement)
 
-   Returns ``True`` if the string *statement* contains one or more complete SQL
-   statements terminated by semicolons. It does not verify that the SQL is
-   syntactically correct, only that there are no unclosed string literals and the
-   statement is terminated by a semicolon.
+   Return ``True`` if the string *statement* appears to contain
+   one or more complete SQL statements.
+   No syntactic verification or parsing of any kind is performed,
+   other than checking that there are no unclosed string literals
+   and the statement is terminated by a semicolon.
 
-   This can be used to build a shell for SQLite, as in the following example:
+   This function may be useful during command-line input
+   to determine if the entered text seems to form a complete SQL statement,
+   or if additional input is needed before calling :meth:`~Cursor.execute`.
 
-   .. literalinclude:: ../includes/sqlite3/complete_statement.py
+   For example::
+
+       >>> sqlite3.complete_statement('SELECT foo FROM bar;')
+       True
+       >>> sqlite3.complete_statement('SELECT foo')
```

Comment:
My mistake for not noticing this when making the original suggestion, but for consistency with the other examples:

```suggestion
       >>> sqlite3.complete_statement("SELECT foo FROM bar;")
       True
       >>> sqlite3.complete_statement("SELECT foo")
```

Reply:
Consistency FTW!